pkgs/by-name/po/portaudio-java/package.nix

```
@@ -0,0 +1,98 @@
+{
+  lib,
+  stdenv,
+  portaudio,
+  jdk,
+  cmake,
+  gradle,
+  fetchFromGitHub,
+  nix-update-script,
+  ...
+}:
+stdenv.mkDerivation rec {
```

```suggestion
stdenv.mkDerivation (finalAttrs: {
```
makes recursion more convenient + allows reusing attrs that aren't set explicitly in this mkDerivation call